libs/atqr/domain/src/lib/interfaces/idsInterfaces.ts

```diff
@@ -0,0 +1,14 @@
+import { Guid } from '@tokilabs/lang';
+
+export interface ChallengeId {
+  challengeId: Guid;
```

Comment:
change the name of the properties because calling for example:
Challenge Id.challengeId doesn't seem very semantic to me.

Reply:
but I will not call ChallengeId. I will implement this interface to type my Challenge prop "id" as challengeId, like that: 

class Challenge implements ChallengeId{
id: challengeId 
}

Reply:
@GabrielaRsanches, I think you misunderstood the intent of the task. This issue was created to temporarily provide the missing types and interfaces required by the rest of the application, and provide TYPE aliases for ids.

Making them interfaces is incorrect. We're NOT supposed to implement these interfaces. See the example: `export type ChallengeId = Guid;`, in this case, we're guaranteeing that ChallengeId has the type of Guid